eng/packaging.targets

```diff
@@ -267,9 +273,15 @@
   </Target>
 
   <Target Name="ValidateAssemblyVersionsInRefPack" 
-          Condition="$(_AssemblyInTargetingPack) == 'true' and '$(PreReleaseVersionLabel)' == 'servicing'" 
+          Condition="'$(SkipValidateAssemblyVersion)' != 'true' and '$(_AssemblyInTargetingPack)' == 'true' and '$(PreReleaseVersionLabel)' == 'servicing'"
           AfterTargets="CoreCompile" >
     <Error Condition="'$(AssemblyVersion)' != '$(LastReleasedStableAssemblyVersion)'" Text="AssemblyVersion should match last released assembly version $(LastReleasedStableAssemblyVersion)" />
```

Comment:
Is this all the validation we're doing @Anipik?  I thought we were going to run a reverse check here: https://github.com/dotnet/runtime/blob/c1e9122f0c3202545aee28243c18b0f33c5c12d9/src/libraries/shims/ApiCompat.proj#L103-L108  
To ensure folks don't add API in servicing.

Reply:
We have an open issue, we haven't enabled it.